superbench/runner/runner.py

```diff
@@ -122,6 +134,17 @@ def __get_mode_command(self, benchmark_name, mode):
                     'superbench.benchmarks.{name}.parameters.distributed_backend=nccl'
                 ).format(name=benchmark_name),
             )
+        elif mode.name == 'mpi':
+            mode_command = (
+                'mpirun '
+                '--allow-run-as-root -hostfile hostfile -map-by ppr:{proc_num}:numa '
```

Comment:
What's meaning of `-map-by`? Would you please all comments to describe the option meaning in the code including `--alow-run-as-root`, `-hostfile`, `-map-by`, etc,?

Reply:
added